Directory.Packages.props

```diff
@@ -7,16 +7,17 @@
     <OTelLatestStableVer>1.5.1</OTelLatestStableVer>
   </PropertyGroup>
 
-  <!-- prod packages -->
+  <!--
```

Comment:
I found that "prod packages" have never been clarified and the current state is messy.
So here I added clarification, also kicked some packages from the "prod" section to the "nonprod" section.